src/stories/molecules/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,35 @@
+// Global Imports
+import React, { ReactNode } from 'react';
+
+// Local Imports
+import { Button } from 'stories/atoms/Button/Button';
+// I know the caret has to wrapped in a wrapper so we can adjust it's placement and allow for roatation, but my first goal was just to render the Caret somewhere on the button first
+// import { DropCaretWrapper } from 'stories/molecules/Dropdown/Dropdown.style';
+// import { SvgContainer } from 'stories/atoms/SVGIcon/svgicon-styles';
+import { ReactComponent as CaretSvg } from 'stories/assets/caret_icon.svg';
+
+interface Props {
+  isMobile?: boolean;
+  // orientation?: string;
+  label?: string;
+  onClick?: () => void;
+  children?: ReactNode; 
+}
+
+export const Dropdown = ({
+  isMobile = false,
```

Comment:
Does this component care about `isMobile` at this time? The designs only show a single style of `Button`. 

And a note on responsiveness, the first approach should always be media queries / regular CSS. An `isMobile` prop   or the use of the `useMediaQuery` custom hook (I think that's what it's called, its been a minute...) should be used only in special circumstances when a regulars CSS media query won't do the trick. We can talk more about this.